llama_parse/base.py

```diff
@@ -158,6 +158,10 @@ class LlamaParse(BasePydanticReader):
         default=False,
         description="Disable the OCR on the document. LlamaParse will only extract the copyable text from the document.",
     )
+    is_formatting_instruction: bool = Field(
+        default=False,
+        description="Allow the parsing instruction to also format the output. Disable to have a cleaner markdown output.",
```

Comment:
`Disable to have a cleaner markdown output.` not sure if well worded as the default is False?
Enable to allow the parsing instruction to have an impact on the output format?